### DISCUSSION TITLE
Web-视差滚动 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#303] [Announcements]
# Web-视差滚动 | Zi-Zi's Journey

转自 B 站 UP 主前端小小陈。

https://gz-metal-cell.github.io/2024/01/13/Web-%E8%A7%86%E5%B7%AE%E6%BB%9A%E5%8A%A8/

**Comment by promefire** [comment]
# 太强了Metal-Cell /(ㄒoㄒ)/~~

**Reply by GZ-Metal-Cell** [reply]
# 看在上帝的份上，我要用我的皮鞋踢你的屁股，蠢货！
